src/components/GetTodos.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from "react";
+import "./Todo.css";
+import { Todo } from "./useFetch";
+import useFetch from "./useFetch";
+
+export default function GetTodos() {
+
+  let {todos,loading,error}=useFetch('http://localhost:8000/todos')
+  const [title, setTitle] = useState<string>("");
+  const [todolist, setTodolist] = useState<Todo[]>([]);
+  const [nextId, setNextId] = useState<number>(1);
+
+  useEffect(()=>{
+    setTodolist(todos)
```

Comment:
set setTodolist once todos are fetched handle this condition.

Reply:
Done